TaskHouseUnitTests/WorkersControllerUnitTests.cs

```diff
@@ -0,0 +1,227 @@
+            Worker worker = null;
+            int id = 2600;
+
+            var result = await controller.Update(id, worker);
+
+            Assert.IsType<BadRequestResult>(result); 
+        }
+
+        [Fact]
+        public async void WorkerController_Update_ReturnsBadRequestResult_WhenIdIsInvalid()
+        { 
+            Worker worker =  new Worker()
+            {
+                Id = 5,
+                Username = "Tusername",
+                Password = "+z490sXHo5u0qsSaxbBqEk9KsJtGqNhD8I8mVBdDJls=", //1234
+                Email = "test@test.com",
+                FirstName = "Bob7",
+                LastName = "Bobsen6",
+                Salt = "upYKQSsrlub5JAID61/6pA=="
+            };
+
+            int id = 2600;
+
+            var result = await controller.Update(id, worker);
+
+            Assert.IsType<BadRequestResult>(result); 
+        }
+
+        [Fact]
+        public async void WorkerController_Delete_ReturnsNoContentResult_WhenIdIsValid()
+        { 
+            Worker worker =  new Worker()
+            {
+                Id = 5,
+                Username = "Tusername",
+                Password = "+z490sXHo5u0qsSaxbBqEk9KsJtGqNhD8I8mVBdDJls=", //1234
+                Email = "test@test.com",
+                FirstName = "Bob7",
+                LastName = "Bobsen6",
+                Salt = "upYKQSsrlub5JAID61/6pA=="
+            };
+
+            int id = 2600;
+            controller.Create(worker);
+            var result = await controller.Delete(worker.Id);
+
+            Assert.IsType<NoContentResult>(result); 
+        }
+
+        [Fact]
+        public async void WorkerController_Update_ReturnsBadRequestResult_WhenWorkerIsNull()
+        { 
+            Worker worker = null;
+            int id = 1234;
+
+            var result = await controller.Update(id, worker);
+
+            Assert.IsType<BadRequestResult>(result); 
+        }
+
+        [Fact]
+        public async void  WorkerController_Delete_ActuallyDeletes_WhenIdIsValid()
+        {
+            Worker TestWorker =  new Worker()
+            {
+                Id = 5,
+                Username = "Tusername",
+                Password = "+z490sXHo5u0qsSaxbBqEk9KsJtGqNhD8I8mVBdDJls=", //1234
+                Email = "test@test.com",
+                FirstName = "Bob7",
+                LastName = "Bobsen6",
+                Salt = "upYKQSsrlub5JAID61/6pA=="
+            };
+
+            await controller.Delete(TestWorker.Id) ;
+            var result = await controller.Get(TestWorker.Id);
+
+            Assert.IsType<NotFoundResult>(result);
+        }
+
+        [Fact] 
+        public async void WorkerController_Delete_ReturnsNotFoundResult_WhenIdIsInvalid()
+        { 
+            int id = 2500; 
+
+            var result = await controller.Delete(id);
+
+            Assert.IsType<NotFoundResult>(result);
+        }
+
+
+        [Fact]
+        public async void WorkerController_Update_ReturnsActionResult_withValidIdAndValidWorker()
```

Comment:
Every outcome will give you an ActionResult - should be 'ReturnsNoContentResult' for better readability of the code